### PR TITLE
Add commands to prompts

### PR DESCRIPTION
**Pull request by sonalideshpandemsft**
Prompt to release should include commands to release independent packages/release groups: [AB#1963](https://dev.azure.com/fluidframework/internal/_sprints/taskboard/Team%20Sasha/internal/CY23Q1/Team%20Sasha%20-%202023.01-A?workitem=1963). 

The function `promptToReleaseDeps` did not include commands which would be displayed when a particular release group such as azure, build-tools, server or client is not released. It was neither included when independent packages such as tinylicious, eslint-config-fluid, protocol-definitions, etc are not released. This PR adds those commands in the `cmd` section. This makes it easier for the release engineer to copy the commands directly in the console.

### CODE REVIEW

**Comment by ssimic2** [comment]
Please update description so it gives enough context to reviewers. It may not be obvious what is "Prompt to release" or what is "independent" in various context. Context/background + clear explanation what you are trying to solve (what issue we face currently that is driving a need for this change), is always very helpful starting point for a review.

File: build-tools/packages/build-cli/src/handlers/promptFunctions.ts [hunk 1]
```diff
@@ -335,6 +335,7 @@ export const promptToReleaseDeps: StateHandlerFunction = async (
             prompt.sections.push({
                 title: "FIRST",
                 message: `Release these packages first:\n\n${chalk.blue(packageSection)}`,
+                cmd: `flub bump deps ${packageSection} --updateType latest`,
```
Comment (tylerbutler) [inline build-tools/packages/build-cli/src/handlers/promptFunctions.ts:338]: The instructions are to release packages, so the commands should be `flub release -g/-p <PACKAGE>`. Maybe instead of having a single step for packages and one for release groups, maybe put each package/release group in its own section. I think that will be clearest to the user.

**Review by tylerbutler — CHANGES_REQUESTED** [review]

File: build-tools/packages/build-cli/packages/runtime/test-runtime-utils/src/assertionShortCodesMap.ts [hunk 2]
```diff
@@ -0,0 +1,10 @@
+/*!
```
Comment (tylerbutler) [inline build-tools/packages/build-cli/packages/runtime/test-runtime-utils/src/assertionShortCodesMap.ts:1]: You need to revert this change.

File: build-tools/packages/build-cli/src/handlers/promptFunctions.ts [hunk 3]
```diff
@@ -331,24 +331,26 @@ export const promptToReleaseDeps: StateHandlerFunction = async (
 			let packageSection = "";
 			for (const [pkg, depVersion] of prereleaseDepNames.packages.entries()) {
 				packageSection += `${pkg} = ${depVersion}`;
+				prompt.sections.push({
+					title: "FIRST",
+					message: `Release these packages first:\n\n${chalk.blue(packageSection)}`,
+					cmd: `${command?.config.bin} ${command?.id} -p ${packageSection}\n`,
```
Comment (tylerbutler) [inline build-tools/packages/build-cli/src/handlers/promptFunctions.ts:337]:
```suggestion
					cmd: `${command?.config.bin} release -p ${pkg}\n`,
```

File: build-tools/packages/build-cli/src/handlers/promptFunctions.ts [hunk 4]
```diff
@@ -331,24 +331,26 @@ export const promptToReleaseDeps: StateHandlerFunction = async (
...
 		if (prereleaseDepNames.releaseGroups.size > 0) {
 			let packageSection = "";
 			for (const [rg, depVersion] of prereleaseDepNames.releaseGroups.entries()) {
 				packageSection += `${rg} = ${depVersion}`;
+				prompt.sections.push({
+					title: "NEXT",
+					message: `Release these release groups:\n\n${chalk.blue(
+						packageSection,
+					)} by running the following command`,
+					cmd: `${command?.config.bin} ${command?.id} -g ${packageSection}\n`,
```
Comment (tylerbutler) [inline build-tools/packages/build-cli/src/handlers/promptFunctions.ts:351]:
```suggestion
					cmd: `${command?.config.bin} release -g ${rg}\n`,
```

**Review by tylerbutler — APPROVED** [review]
Couple of minor things.

File: build-tools/packages/build-cli/src/handlers/promptFunctions.ts [hunk 5]
```diff
@@ -331,24 +331,26 @@ export const promptToReleaseDeps: StateHandlerFunction = async (
 			let packageSection = "";
 			for (const [pkg, depVersion] of prereleaseDepNames.packages.entries()) {
 				packageSection += `${pkg} = ${depVersion}`;
```
Comment (tylerbutler) [inline build-tools/packages/build-cli/src/handlers/promptFunctions.ts:333]:
```suggestion
				packageSection = `${pkg} = ${depVersion}`;
```

File: build-tools/packages/build-cli/src/handlers/promptFunctions.ts [hunk 6]
```diff
@@ -331,24 +331,26 @@ export const promptToReleaseDeps: StateHandlerFunction = async (
...
-			prompt.sections.push({
-				title: "FIRST",
-				message: `Release these packages first:\n\n${chalk.blue(packageSection)}`,
-			});
 		}
 
 		if (prereleaseDepNames.releaseGroups.size > 0) {
 			let packageSection = "";
 			for (const [rg, depVersion] of prereleaseDepNames.releaseGroups.entries()) {
 				packageSection += `${rg} = ${depVersion}`;
```
Comment (tylerbutler) [inline build-tools/packages/build-cli/src/handlers/promptFunctions.ts:345]:
```suggestion
				packageSection = `${rg} = ${depVersion}`;
```

File: build-tools/packages/build-cli/src/handlers/promptFunctions.ts [hunk 7]
```diff
@@ -331,24 +331,26 @@ export const promptToReleaseDeps: StateHandlerFunction = async (
 			let packageSection = "";
 			for (const [pkg, depVersion] of prereleaseDepNames.packages.entries()) {
 				packageSection += `${pkg} = ${depVersion}`;
+				prompt.sections.push({
+					title: "FIRST",
```
Comment (tylerbutler) [inline build-tools/packages/build-cli/src/handlers/promptFunctions.ts:335]:
```suggestion
					title: "RELEASE PACKAGE",
```

File: build-tools/packages/build-cli/src/handlers/promptFunctions.ts [hunk 8]
```diff
@@ -331,24 +331,26 @@ export const promptToReleaseDeps: StateHandlerFunction = async (
...
-				message: `Release these packages first:\n\n${chalk.blue(packageSection)}`,
-			});
 		}
 
 		if (prereleaseDepNames.releaseGroups.size > 0) {
 			let packageSection = "";
 			for (const [rg, depVersion] of prereleaseDepNames.releaseGroups.entries()) {
 				packageSection += `${rg} = ${depVersion}`;
+				prompt.sections.push({
+					title: "NEXT",
```
Comment (tylerbutler) [inline build-tools/packages/build-cli/src/handlers/promptFunctions.ts:347]:
```suggestion
					title: "RELEASE RELEASE GROUP",
```
